plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteMetadata.java

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
Please add below code and set `SUPPORTS_SET_COLUMN_TYPE` as false in Ignite connector test. This fixes CI failure.
```java
    @Override
    public void setColumnType(ConnectorSession session, JdbcTableHandle handle, JdbcColumnHandle column, Type type)
    {
        throw new TrinoException(NOT_SUPPORTED, "This connector does not support setting column types");
    }
```

Reply:
Thank you!